binaries/daemon/src/lib.rs

```diff
@@ -314,6 +320,9 @@ impl Daemon {
                 tracing::warn!("failed to update HLC with incoming event timestamp: {err}");
             }
 
+            let start = Instant::now();
+            let event_debug = format!("{inner:?}");
+
```

Comment:
What is event_debug for?

Reply:
This is also used for the warning mentioned [above](https://github.com/dora-rs/dora/pull/901#discussion_r2056304232). We should probably make this cheaper, though. I just pushed 20a1ade6 to use just the variant name instead of the full debug representation (to avoid performance overhead).